test/junit/scala/util/UsingTest.scala

```diff
@@ -230,12 +230,12 @@ class UsingTest {
 
   @Test
   def safeUsingMultipleResourcesPropagatesCorrectlySimple(): Unit = {
-    val scala.util.Failure(usingException) = for {
-      _ <- Using(new ExceptionResource)
-      _ <- Using(new ErrorResource)
-    } yield {
-      throw new UsingException("nested `Using`")
-    }
+    val scala.util.Failure(usingException) =
+      Using(new ExceptionResource) { _ =>
+        Using(new ErrorResource) { _ =>
+          throw new UsingException("nested `Using`")
+        }
```

Comment:
missing `.get` here, which is exactly the reason why `map` and `flatMap` existed in the first place. I'm not happy with this solution at all.